yb-voyager/cmd/importData.go

```diff
@@ -164,11 +160,7 @@ func getYBServers() []*tgtdb.Target {
 
 			// check if given host is one of the server in cluster
 			if loadBalancerUsed {
-				for _, ip := range potentialLBIps {
-					if ip.String() == host || ip.String() == public_ip {
-						loadBalancerUsed = false
-					}
-				}
+				tryMatchingWithTargetHost(host, public_ip)
```

Comment:
Rename the function as `isSeedHost(host, publicIP)`.
If the function returns true, set `loadBalancerUsed = false`.